SwiftyAttributes/Sources/common/NSString+SwiftyAttributes.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  NSString+SwiftyAttributes.swift
+//  SwiftyAttributes
+//
+//  Created by Roman Podymov on 26/05/19.
+//  Copyright © 2019 Roman Podymov. All rights reserved.
+//
+
+#if os(macOS)
+    import AppKit
+#else
+    import UIKit
+#endif
+
+extension NSString {
+    
+    /**
+     Get string size with the specified attributes.
+     
+     - parameter    attrs:   The attributes to use.
+     */
+    public func size(withSwiftyAttributes attrs: [Attribute]? = nil) -> Size {
```

Comment:
I'd leave out the default parameter here because then `size()` will be ambiguous.

Reply:
Hello @eddiekaiger 
What about the `swifty_` prefix for all these functions?